src/main/java/uk/gov/justice/hmpps/offenderevents/services/EventRetrievalService.java

```diff
@@ -56,8 +56,8 @@ public void pollEvents(final LocalDateTime currentTime) {
         repository.save(audit);
 
         final var startTime = audit.getNextStartTime();
-        final var timeDifferenceHrs = startTime.until(latestPollTimeAllowed, ChronoUnit.HOURS);
-        final var endTime = timeDifferenceHrs > maxEventRangeHours ? startTime.plusHours(maxEventRangeHours) : latestPollTimeAllowed;
+        final var timeDifferenceMins = startTime.until(latestPollTimeAllowed, ChronoUnit.MINUTES);
```

Comment:
I did work out why we the logs were saying
```
Getting events between 2020-03-24T14:01:23.551732 and 2020-03-24T15:37:52.874706
```
the method `until` will say that those two times are only an hour apart (rounded down to the nearest hour) so did therefore allow the two times to be 1:59 apart

Reply:
Yes!!